### DISCUSSION TITLE
Grey screen after install

### DISCUSSION THREAD

**Discussion by Adithyan007hack** [Matt-FTW/dotfiles#10] [Q&A]
I have installed , your dotfile configuration and hyprland in Arch linux , everything went fine during installation but after login in
only grey screen in showing with a cursor is there any fix to the issue  and loging in via sddm cause it to loop back to sddm

**Comment by Matt-FTW** [comment]
Hi, 
So I think that your problem is related to the new swww update. I have the same issue from 0.9.0 onwards. 
Execute this command and tell me what the output is
```bash
swww --version
```
If its 0.9.0 or any other thats not lower, try downgrading the package to version 0.8.2.

**Reply by Adithyan007hack** [reply]
Command not found 
how do I install swww version 0.8.2

**Reply by Matt-FTW** [reply]
You have to download the PKGBUILD for the version from the AUR to compile from source.
Here's the file: https://aur.archlinux.org/cgit/aur.git/commit/?h=swww&id=f5971656d37d7ded2e803f9655685c416ca6b71e
Download the tar, extract the files and once you're into the directory, write the following command to execute the PKGBUILD:
```bash
makepkg -si
```
